app/views/episodes/show.slim

```diff
@@ -1,3 +1,16 @@
 h1 = @episode.title
 br
 iframe src = youtube_video_link(@episode.video) width = '560' height = '315' frameborder = '0' allow = 'accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture' allowfullscreen
+
+br
+- if can? :manage, Episode
+    = link_to 'Edit', edit_episode_path(@episode)
+    br
+    = "Начало подкаста: #{@episode.broadcast_begin}"
+    br
+    = "Конец подкаста: #{@episode.broadcast_end}"
+    br
+    = "Статус подкаста: #{@episode.youtube_status}"
+    br
+    = button_to 'Update youtube info', "/update_youtube/#{@episode.id}", :method => :post
```

Comment:
Use new [hash syntax](http://effectif.com/ruby/update-your-project-for-ruby-19-hash-syntax)